ops/terraform/modules/resources/bfd_monitoring/main.tf

```diff
@@ -0,0 +1,98 @@
+locals {
+  log_groups = {
+    access = "/bfd/${var.env}/bfd-server/access.json"
+  }
+  namespace = "json/bfd-${var.env}/bfd-server"
```

Comment:
Just want to recap what was discussed for deployment. 
Metric Filters would be applied to existing namespace & prepended json/ would be dropped from metric names.